random.cabal

```diff
@@ -63,14 +63,26 @@ build-type:         Custom
 extra-source-files:
     README.md
     CHANGELOG.md
+tested-with:         GHC == 7.10.2
+                   , GHC == 7.10.3
+                   , GHC == 8.0.2
+                   , GHC == 8.2.2
+                   , GHC == 8.4.3
```

Comment:
```suggestion
```

8.4.3 is not enabled in `.travis.yml`.

Reply:
It should be, which means that is not the correct change.

Reply:
Ah right, thanks. I guess once this PR and https://github.com/idontgetoutmuch/random/pull/148/files#diff-bb794c45fcbccfb6ea3e4c9ca0122833 are in, the `tested-with` list and the `.travis.yml` will be in sync.